src/js/components/Tip/stories/typescript/Simple.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useRef } from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { grommet, Box, Button, Grommet, Text } from 'grommet';
+import { Tip } from '../../Tip';
+
+const Example = () => {
+  const ref = useRef();
+  return (
+    <Grommet full theme={grommet}>
+      <Box align="center" justify="center" background="dark-1" fill>
+        <Tip
+          content={
+            <Box
```

Comment:
Can we make the "Simple" story even simpler?
```
<Tip content="something helpful"><Button label="action" /></Tip>
```

Reply:
Done.